src/pages/en/tutorial/4-layouts/3.md

```diff
@@ -78,15 +80,47 @@ You already have a `BaseLayout.astro` for defining the overall layout of your pa
 
 ### Test your knowledge
 
-Use these words in the text below. Click the blanks to reveal if you're right!
-
-<Blanks>
-- blog
-- `BaseLayout`
-- component-based design
-</Blanks>
-
-Because Astro uses || **component-based design** ||, you can nest one layout inside another and take advantage of working with modular pieces. This is particularly helpful when creating a || **blog** ||, so that a || **`BaseLayout`** || can just concern itself with main page styling, while a `MarkdownPostLayout` can focus on specific templating that is only used for your Markdown posts. 
+1. This allows you to you nest one layout inside another and take advantage of working with modular pieces.
+
+  <MultipleChoice>
+    <Option>
+      blog
+    </Option>
+    <Option>
+      `BaseLayout.astro`
+    </Option>
+    <Option isCorrect>
+       component-based design
+    </Option>
+  </MultipleChoice>
+
+2. Multiple layouts are particularly useful for projects that contain Markdown pages, like a...
+
+  <MultipleChoice>
+    <Option isCorrect>
+      blog
+    </Option>
+    <Option>
+      `BaseLayout.astro`
+    </Option>
+    <Option>
+       component-based design
+    </Option>
+  </MultipleChoice>
+
+3. This can just concern itself with main page styling, while `MarkdownPostLayout.astro` can focus on specific templating that is only used for your Markdown posts.
```

Comment:
This question seems kind of wordy and hard to understand (Hemingway App makes it RED 😱). On first read I was like “This” what? And then understood “This” means the answer.

Is this _too_ extreme in the other direction?

```suggestion
3. Which of these provides templating for all your pages?
```